pkg/client/list_test.go

```diff
@@ -64,9 +64,19 @@ func TestGetVolumeList(t *testing.T) {
 	}
 
 	objects := []runtime.Object{}
-	for i := 0; i < 2000; i++ {
```

Comment:
value 2000 is used to test whether we ae getting mre than 1000 objects. You should add new tests not by replacing existing tests.

Reply:
done